### ISSUE TITLE
PSR4

### ISSUE THREAD

**Issue by banago** [SimonEast/Simple-Ansi-Colors#1]
Hi @SimonEast,

Do you think you can make the repo a PSR4. I'd like to grab this through composer over at PHPloy rather then manually include in the repo.

Let me know.

**Comment by SimonEast** [comment]
Yeah, composer compatibility would be great.  Are you able to submit a pull-request, or at least help me understand what changes are needed to make this possible?

**Comment by SimonEast** [comment]
Actually, I just realised that it should already support composer via PSR-0, but not PSR-4, so just need to know what changes are needed for that.
